e2e/addons/addons_test.go

```diff
@@ -47,6 +48,18 @@ var _ = Describe("addons flow", func() {
 			Expect(appShowOutput.Name).To(Equal(appName))
 			Expect(appShowOutput.URI).To(BeEmpty())
 		})
+
+		It("app delete does delete .workspace", func() {
+			_, err := cli.AppDelete(map[string]string{"": ""})
+			Expect(err).NotTo(HaveOccurred())
+			Expect("./copilot").Should(BeADirectory())
+			Expect("./copilot/.workspace").ShouldNot(BeAnExistingFile())
+
+			_, initErr = cli.AppInit(&client.AppInitRequest{
+				AppName: appName,
+			})
+			Expect(initErr).NotTo(HaveOccurred())
+		})
```

Comment:
I actually want to move this block of code for last and check to see if the manifest.yml and addons/ are still present. Will send another PR out